visualization/app/codeCharta/ui/codeMap/threeViewer/threeSceneService.ts

```diff
@@ -126,4 +149,9 @@ export class ThreeSceneService {
 	public getHighlightedBuilding(): CodeMapBuilding {
 		return this.highlighted
 	}
+
+	private reselectBuilding() {
+		const buildingToSelect: CodeMapBuilding = this.getMapMesh().getBuildingByPath(this.selected.node.path)
+		this.selectBuilding(buildingToSelect)
```

Comment:
call this when a building was found (shouldnt find one when it's hidden or excluded